contracts/libraries/JBFundingCycleMetadataResolver.sol

```diff
@@ -112,26 +108,26 @@ library JBFundingCycleMetadataResolver {
 
   /// @notice Pack the funding cycle metadata.
   /// @param _metadata The metadata to validate and pack.
-  /// @return packed The packed uint256 of all metadata params. The first 8 bits specify the version.  
+  /// @return packed The packed uint256 of all metadata params. The first 8 bits specify the version.
   function packFundingCycleMetadata(JBFundingCycleMetadata memory _metadata)
     internal
     pure
     returns (uint256 packed)
   {
     // version 1 in the bits 0-7 (8 bits).
     packed = 1;
```

Comment:
Curious how version could be used but don't really understand, ballot redemption removal still look good below this tho.